cli/src/main.rs

```diff
@@ -52,6 +54,225 @@ pub fn main() {
+        lwd_bin.push("lightwalletd");
+
+        if !std::path::Path::is_file(lwd_bin.as_path()) {
+            panic!("can't find lwd bin! exiting.");
+        }
+
+        let lwd_confs: PathBuf = [
+            worktree_home.clone(),
+            OsString::from("regtest"),
+            OsString::from("conf"),
+            OsString::from(""),
+        ]
+        .iter()
+        .collect();
+
+        let mut unflagged_lwd_conf: String = String::new();
+        unflagged_lwd_conf.push_str(lwd_confs.to_str().expect("trouble making flagged_lwd_conf"));
+        unflagged_lwd_conf.push_str("lightwalletdconf.yml");
+
+        // for lwd config
+        let mut unflagged_zcashd_conf: String = String::new();
+        unflagged_zcashd_conf.push_str(
+            zcash_confs
+                .to_str()
+                .expect("error making unflagged zcash conf"),
+        );
+        unflagged_zcashd_conf.push_str("zcash.conf");
+
+        let lwd_datadir: PathBuf = [
+            worktree_home.clone(),
+            OsString::from("regtest"),
+            OsString::from("datadir"),
+            OsString::from("lightwalletd"),
+            OsString::from(""),
+        ]
+        .iter()
+        .collect();
+
+        let mut unflagged_lwd_datadir: String = String::new();
+        unflagged_lwd_datadir.push_str(lwd_datadir.to_str().expect("error making lwd_datadir"));
+
+        let lwd_logs: PathBuf = [
+            worktree_home.clone(),
+            OsString::from("regtest"),
+            OsString::from("logs"),
+            OsString::from(""),
+        ]
+        .iter()
+        .collect();
+        let mut unflagged_lwd_log: String = String::new();
+        unflagged_lwd_log.push_str(lwd_logs.to_str().expect("error making lwd_datadir"));
+        unflagged_lwd_log.push_str("lwd.log");
+
+        let lwd_command = Command::new(lwd_bin)
+            .args([
+                "--no-tls-very-insecure",
+                "--zcash-conf-path",
+                &unflagged_zcashd_conf,
+                "--config",
+                &unflagged_lwd_conf,
+                "--data-dir",
+                &unflagged_lwd_datadir,
+                "--log-file",
+                &unflagged_lwd_log,
+            ])
+            // this will print stdout of lwd process' output also to the zingo-cli stdout
+            .stdout(Stdio::inherit())
+            .stderr(Stdio::inherit())
+            .spawn()
+            .expect("failed to start lwd");
+
+        // this client's stdout is less verbose so logging it may not be needed along with the working lwd.log file.
+        //let mut lwd_stdout_logfile =
+        //File::create("/zingolib/regtest/logs/lwd-ping.log").unwrap();
+        // the next line is a halting process..
+        //io::copy(&mut lwd_command.stdout.unwrap(), &mut lwd_stdout_logfile).unwrap();
+
+        // wait 5 seconds for lwd to fire up
+        // very generous, plan to tune down
+        let five_seconds = time::Duration::from_millis(5_000);
```

Comment:
Again...   why sleep?   Let's find out what happens when these processes (`zcashd`, `lightwalletd`, `zingo-cli`) don't have their counterparts on launch...   they should do something reasonable.   We should enforce those reasonable behaviors.